# 机器人状态消息
std_msgs/Header header
# 关节位置
JointPosition joint_position

# TCP位姿
TCPPose tcp_pose

# 是否完成运动
bool motion_done

# 错误码
int32 error_code

